test/statetest/statetest_loader.cpp

```diff
@@ -271,9 +271,9 @@ state::State from_json<state::State>(const json::json& j)
         {
             for (const auto& [j_key, j_value] : storage_it->items())
             {
-                const auto value = from_json<bytes32>(j_value);
-                acc.storage.insert(
-                    {from_json<bytes32>(j_key), {.current = value, .original = value}});
+                if (const auto value = from_json<bytes32>(j_value); value)
```

Comment:
could better be for readability
```suggestion
                if (const auto value = from_json<bytes32>(j_value); !is_zero(value))
```

Reply:
How about this?

```suggestion
                if (const auto value = from_json<bytes32>(j_value))
```

Reply:
fine for me